TestSwitchApi/TestSwitchApi/Models/Response/ListResponse.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Collections.Generic;
+using TestSwitchApi.Models.Request;
+
+namespace TestSwitchApi.Models.Response
+{
+   public class ListResponse<T>
+    {
+        private readonly string _path;
+        public IEnumerable<T> Items { get; }
+        public int TotalNumberOfItems { get; }
+        public int Page { get; }
+        public int PageSize { get; }
+
+        public string NextPage
+        {
+            get
+            {
+                if (!HasNextPage())
+                {
+                    return null;
+                }
+
+                var url = $"/{_path}?page={Page + 1}&pageNumber={PageSize}";
+
+                return url;
+            }
+        }
+
+        public string PreviousPage
+        {
+            get
+            {
+                if (Page <= 1)
+                {
+                    return null;
+                }
+
+                var url = $"/{_path}?page={Page - 1}&pageNumber={PageSize}";
+
```

Comment:
same here - could we add a couple of very quick unit tests to check this behaviour?